datasets/code_x_glue_cc_clone_detection_big_clone_bench/README.md

```diff
@@ -0,0 +1,152 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- C++
+licenses:
+- other-C-UDA
+multilinguality:
+- monolingual
+size_categories:
+- n>1M
+source_datasets: []
+task_categories:
+- text-classification
+task_ids:
+- semantic-similarity-classification
+---
+# Dataset Card for "code_x_glue_cc_clone_detection_big_clone_bench"
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
```

Comment:
Can you re-add all the sections that were removed ?

You can leave their content with `[More Information Needed]`

Reply:
For the names, I took the original names, but I understand.
For this one, 'cxgcc_clone_detection_big' would be ok ? (cxg for CodeXGlue, cc for code to code, and a shorten version of the dataset name (and same thing for the others).
For the "make style", it was part of my post-processing script, but there may some subtlety somewhere.

Reply:
(and I will re-add the missing sections)

Reply:
(I just ran make style, and there are no changes to the code, can you confirm that there is an issue with the code style ?)

Reply:
Here a proposal list of names: 

cxgcc_defect_detect
cxgcc_clone_detect_big
cxgcc_refine
cxgcc_complete_token
cxgcc_code_to_code
cxgcc_code_complete_line
cxgcc_clone_detect_poj
cxgcc_cloze_test_maxmin
cxgcc_cloze_test_all
cxgct_code_to_text
cxgtt_text_to_text
cxgtc_search_web_query
cxgtc_text_to_code
cxgtc_search_adv

Reply:
Looks good !
You can even make the name clearer with
```
code_xglue_<tt|cc|ct|tc>_<subdataset_name>
```
(just change the beginning from `cxg` to `code_xglue_`)

Reply:
The remaining code style issues are these ones:
```
src/datasets/commands/dummy_data.py:132:25: E722 do not use bare 'except'
src/datasets/commands/dummy_data.py:141:21: F841 local variable 'e' is assigned to but never used
```
So all the code in your datasets is fine, the only errors come from `dummy_data.py`